src/som/interpreter/actors/EventualSendNode.java

```diff
@@ -256,5 +261,12 @@ protected boolean isTaggedWith(final Class<?> tag) {
       }
       return super.isTaggedWith(tag);
     }
+
+    protected boolean isMessageBreakpointed(final Actor receiver, final RootCallTarget root) {
+      if (receiver instanceof LocalManager) {
```

Comment:
The test here should be on `VmSettings.TRUFFLE_DEBUGGER_ENABLED` instead.
The main reason is that the VmSettings field is a constant, which can be easily resolved by the compiler, so that the test does not have to be done at runtime in the non-debugging mode. This is very important for performance.